Added video of IO venue

DarshilBhatt opened the pull request.

Commit:
Added video of IO venue

manan-jadhav commented on _posts/2018-07-18-io-extended.md:
```diff
@@ -5,6 +5,7 @@ date: 2018-07-18
 categories: ["Events"]
 tags: ["Google IO", "IO 2018 Extended"]
 author: Darshil Bhatt
+assets: "/assets/2018-07/Google IO 2018 Extended"
```
Dont need this really

manan-jadhav commented on _posts/2018-07-18-io-extended.md:
```diff
@@ -16,7 +17,7 @@ Before live streaming started, some members of community delivered the talks on
 * Google Cloud Platform
 
-We enjoyed seeing the video of Google IO venue which was made by the two co-organizers of GDG Baroda who were at Shoreline.That was just an awesome! Finally,the live streaming started with the rocking keynote by Sundar Pichai. It was a feeling like as we are actually attending the event at Shoreline itself.We were just enjoying the keynote in the anticipation of products that are coming up.Sundar ...
+We enjoyed seeing the [video](https://photos.app.goo.gl/XsnBjAXxPFttZwfz7) of Google IO venue which was made by the two co-organizers of GDG Baroda who were at Shoreline.That was just an awesome! Finally,the live streaming started with the rocking keynote by Sundar Pichai. It was a feeling like as we are actually attending the event at Shoreline itself.We were just enjoying the keynote in the ant ...
```
How about embedding the video instead of linking to it?

manan-jadhav reviewed (changes requested):
Please add images of the extended event

DarshilBhatt commented:
Removed assets column, added images & video embedded.

On 21 Jul 2018 5:26 p.m., "Manan Jadhav" <notifications@github.com> wrote:

> *@CurosMJ* requested changes on this pull request.
>
> Please add images of the extended event
> ------------------------------
>
> In _posts/2018-07-18-io-extended.md
> <https://github.com/gdgbaroda/gdgbaroda.github.io/pull/21#discussion_r204209145>
> :
>
> > @@ -5,6 +5,7 @@ date: 2018-07-18
>  categories: ["Events"]
>  tags: ["Google IO", "IO 2018 Extended"]
>  author: Darshil Bhatt
> +assets: "/assets/2018-07/Google IO 2018 Extended"
>
> Dont need this really
> ------------------------------
>
> In _posts/2018-07-18-io-extended.md
> <https://github.com/gdgbaroda/gdgbaroda.github.io/pull/21#discussion_r204209158>
> :
>
> > @@ -16,7 +17,7 @@ Before live streaming started, some members of community delivered the talks on
>  * Google Maps
>  * Google Cloud Platform
>
> -We enjoyed seeing the video of Google IO venue which was made by the two co-organizers of GDG Baroda who were at Shoreline.That was just an awesome! Finally,the live streaming started with the rocking keynote by Sundar Pichai. It was a feeling like as we are actually attending the event at Shoreline itself.We were just enjoying the keynote in the anticipation of products that are coming up.Sundar Pichai & other Google leads were speaking about the coming products of Google.This time in IO, the big announcements in the Google technologies were:-
> +We enjoyed seeing the [video](https://photos.app.goo.gl/XsnBjAXxPFttZwfz7) of Google IO venue which was made by the two co-organizers of GDG Baroda who were at Shoreline.That was just an awesome! Finally,the live streaming started with the rocking keynote by Sundar Pichai. It was a feeling like as we are actually attending the event at Shoreline itself.We were just enjoying the keynote in the anticipation of products that are coming up.Sundar Pichai & other Google leads were speaking about the coming products of Google.This time in IO, the big announcements in the Google technologies were:-
>
> How about embedding the video instead of linking to it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gdgbaroda/gdgbaroda.github.io/pull/21#pullrequestreview-139256405>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AfE7PPBiok8asNYPJhmYda0UaDqjI47uks5uIxbygaJpZM4VZjEq>
> .
>
